src/supervisor/watchers/handlers/index.ts

```diff
@@ -23,11 +23,6 @@ async function isSupportedNamespacedWorkload(
       return await deploymentConfig.isNamespacedDeploymentConfigSupported(
         namespace,
       );
-    case WorkloadKind.CronJobV1Beta1:
-      return await cronJob.isNamespacedCronJobSupported(
-        workloadKind,
-        namespace,
-      );
     case WorkloadKind.CronJob:
```

Comment:
can also remove this case as the switch defaults to true (and cronjobs should always be supported now)